tests/Microsoft.DotNet.Docker.Tests/Manifest.cs

```diff
@@ -0,0 +1,34 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+
+namespace Microsoft.DotNet.Docker.Tests
+{
+    // Simplified version of the ImageBuilder manifest model
+    // See https://github.com/dotnet/docker-tools/tree/main/src/Microsoft.DotNet.ImageBuilder/src/Models/Manifest
+    public class Manifest
+    {
+        public List<Repo> Repos { get; set; } = new List<Repo>();
+    }
+
+    public class Repo
```

Comment:
Consider using Records for these model classes, as they have automatically implemented ToString() methods - helps with logging and debugging.

```suggestion
    public record Repo
```

Reply:
I tried this, and Json deserialization did not work as expected. After doing a little research, I learned that I'd have to write additional constructors in order for the deserialization to work. That feels a bit unnecessary, so I'm choosing to leave them as classes.